src/oss-tests/ProjectManagerTests/PyPIProjectManagerTests.cs

```diff
@@ -147,7 +147,20 @@ public async Task EnumerateVersionsSucceeds(string purlString, int count, string
             Assert.AreEqual(count, versions.Count);
             Assert.AreEqual(latestVersion, versions.First());
         }
-        
+
+        [DataTestMethod]
+        [DataRow("pkg:pypi/pandas@1.4.2")]
+        [DataRow("pkg:pypi/plotly@3.7.1")]
+        [DataRow("pkg:pypi/requests@2.27.1")]
+        public async Task PackageVersionExistsAsyncSucceeds(string purlString)
+        {
+            PyPIProjectManager projectManager = new(".");
```

Comment:
You shouldn't have to create a new projectManager. Should be able to use `_projectManager`

Reply:
Sure, I can also use _projectManager as long as I set up the http responses with the mock handler at the top of file. I wasn't sure if we wanted to do that because I noticed NPMProjectManager.PackageVersionExistsAsyncSucceeds() creates a new project manager. Do we want to follow the same pattern or are we good to diverge?